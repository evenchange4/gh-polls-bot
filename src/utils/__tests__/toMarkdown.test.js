// @flow
const toMarkdown = require('../toMarkdown');

it('should return markdown', () => {
  expect(toMarkdown('MOCK_ID')(['1', '2', '3'])).toMatchSnapshot();
});

it('should return markdown with only one option', () => {
  expect(toMarkdown('MOCK_ID')(['1'])).toMatchSnapshot();
});
