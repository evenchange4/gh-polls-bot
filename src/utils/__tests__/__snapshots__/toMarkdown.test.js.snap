// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return markdown 1`] = `
"[![](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/1)](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/1/vote)
[![](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/2)](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/2/vote)
[![](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/3)](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/3/vote)"
`;

exports[`should return markdown with only one option 1`] = `"[![](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/1)](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/MOCK_ID/1/vote)"`;
